model-optimizer/extensions/front/kaldi/replace_dropoutmask.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from extensions.middle.MakeKaldiConstReshapable import create_const_with_batch_from_input
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.graph.graph import Graph
+
+
+class ReplaceDropoutMaskPattern(FrontReplacementPattern):
+    enabled = True
+    run_non_recursive = True
```

Comment:
`run_non_recursive` is incorrect member. I think we don't have body graphs in Kaldi. So this attribute is not needed

Reply:
agree